### PR TITLE
latest build_verify

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: webdev/test/build/ensure_build_test.dart [hunk 1]
```diff
@@ -12,6 +10,9 @@ import 'package:build_verify/build_verify.dart';
 import 'package:test/test.dart';
 
 void main() {
-  test('ensure_build',
-      () => expectBuildClean(packageRelativeDirectory: 'webdev'));
+  test(
+    'ensure_build',
+    () => expectBuildClean(packageRelativeDirectory: 'webdev'),
+    timeout: const Timeout.factor(2),
```
Comment (nshahan) [inline webdev/test/build/ensure_build_test.dart:16]: Is this doubling the timeout? Is that needed for some reason?

**Review by nshahan — APPROVED** [review]
LGTM with one question about the timeout.

File: webdev/test/build/ensure_build_test.dart (hunk 1 above)
Comment (kevmoo) [reply webdev/test/build/ensure_build_test.dart:16]: The new build_verify can be slower...that's all

**Review by annagrin — APPROVED** [review]
LGTM!

**Pull request merged**